### PR TITLE
Add link to Service Fabric Team tour video

### PR DESCRIPTION
**Pull request by GitTorre**
Meet the Team section at bottom of root readme.md

### CODE REVIEW

**Comment by bpm-ms** [comment]
This is pretty cool, but I'm wondering if the singular video which has a partial impromptu tour of the office is substantial enough to be a major header in the initial README.  We will want to reduce clutter and have top level / essential items here.  I think a more compelling section to loop in the videos could be a playlist that we continuously add to, and inside a heading of something like "Inside the Cockpit" or "Check out what's going on behind the scenes", and link to all of the videos we will have as opposed to a singular video.  

Maybe we could have something like this?
# Additional Resources
## Behind the Scenes
[See our developers at work and get to know us!](Link to youtube playlist)

---

Just my thoughts/opinion, not blocking the PR for this or asking for changes

**Comment by GitTorre** [comment]
Agreed. Unfortunately, it’s not clear to me how we link to Playlists on YouTube. This, the link to part 1, which will contain links to other parts. In some sense, the top level video should be an introduction to the project, with useful information on high level design and architecture of the product. Great feedback.

**Comment by anmolah** [comment]
+1 on @bpm-ms thoughts. I dont think this should be on the main page. We can chat on how to fit it into our docs but I dont have a clear answer yet.

**Comment by rishirsinha** [comment]
One proposal could be to start off two new pages:

1. About the team.
2. About the technology.

The first link would go to set of videos and links to linked-in profiles for the members of the team (members related to the project not necessarily product).

The second link would go into the various white papers about the technology.

**Comment by GitTorre** [comment]
Here is a link to the Behind the Product playlist, including an informal chat with Anmol and Vac covering our move to OSS.

https://www.youtube.com/playlist?list=PLlrxD0HtieHh73JryJJ-GWcUtrqpcg2Pb

...Charles

File: README.md [hunk 1]
```diff
@@ -130,11 +130,12 @@ For more details please refer [Deploying local cluster from build](docs/install_
 Service Fabric conceptual and reference documentation is available at [docs.microsoft.com/azure/service-fabric](https://docs.microsoft.com/azure/service-fabric/). Documentation is also open to your contribution on GitHub at [github.com/Microsoft/azure-docs](https://github.com/Microsoft/azure-docs).
 ## Samples 
 For Service Fabric sample code, check out the [Azure Code Sample gallery](https://azure.microsoft.com/resources/samples/?service=service-fabric) or go straight to [Azure-Samples on GitHub](https://github.com/Azure-Samples?q=service-fabric).
+## Service Fabric - Behind the Curtain  
+<a href="https://www.youtube.com/playlist?list=PLlrxD0HtieHh73JryJJ-GWcUtrqpcg2Pb&disable_polymer=true"><strong>Take a virtual tour with us</strong></a> and meet some of the folks who design and implement service fabric. This playlist will grow over time. Check back often!
```
Comment (anmolah) [inline README.md]: This playlist will grow over time with content describing the inner workings of the product.

You think we should add the above ?

**Pull request merged**